CMakeLists.txt

```diff
@@ -197,6 +197,7 @@ elseif(MSVC)
       "C4061" # enumerator 'identifier' in switch of enum 'enumeration' is not
               # explicitly handled by a case label
               # Disable this because it flags even when there is a default.
+      "C4091" # 'keyword' : ignored on left of 'type' when no variable is declared
```

Comment:
Do you have an example of one of these errors? Is this from new code?

Reply:
This is not new code, which exists in previous repo. I forgot to migrate this line to new repo.

Reply:
Below is the example of warning. The reported lines are not aws-lc code.
```
"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\cl.exe"  /nologo /TP -DBORINGSSL_IMPLEMENTATION -DBORINGSSL_RELEASE_BUILD -DNOMINMAX -DWIN32_LEAN_AND_MEAN -D_CRT_SECURE_NO_WARNINGS -D_HAS_EXCEPTIONS=0 -D_STL_EXTRA_DISABLED_WARNINGS="4774 4987" -I..\third_party\googletest\in
clude -I..\crypto\..\include -utf-8 -Wall -WX  -wd4061 -wd4100 -wd4127 -wd4200 -wd4204 -wd4221 -wd4242 -wd4244 -wd4267 -wd4371 -wd4388 -wd4296 -wd4350 -wd4365 -wd4389 -wd4464 -wd4510 -wd4512 -wd4514 -wd4548 -wd4610 -wd4623 -wd4625 -wd4626 -wd4628 -wd4668 -wd4706 -wd4710 -wd4711 -wd4800 -wd48
20 -wd5026 -wd5027 -wd5045  -w44265 /MD /O2 /Ob2 /DNDEBUG /showIncludes /Focrypto\test\CMakeFiles\test_support_lib.dir\abi_test.cc.obj /Fdcrypto\test\CMakeFiles\test_support_lib.dir\test_support_lib.pdb /FS -c ..\crypto\test\abi_test.cc
C:\Program Files (x86)\Windows Kits\8.1\include\um\dbghelp.h(1544): error C2220: warning treated as error - no 'object' file generated
C:\Program Files (x86)\Windows Kits\8.1\include\um\dbghelp.h(1544): warning C4091: 'typedef ': ignored on left of '' when no variable is declared
C:\Program Files (x86)\Windows Kits\8.1\include\um\dbghelp.h(3190): warning C4091: 'typedef ': ignored on left of '' when no variable is declared
```